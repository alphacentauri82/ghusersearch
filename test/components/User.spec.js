import React from 'react';
import renderer from 'react-test-renderer';
import {
  StaticRouter as Router,
} from 'react-router-dom';
import {
  StyleSheetTestUtils,
} from 'aphrodite';

import User from 'components/User';

describe('Component: User', () => {

  beforeAll(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterAll(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('should render', () => {
    const component = renderer.create(
      <Router context={{}}>
        <User
          login="simon"
          avatar_url="url"
        />
      </Router>
    );
    expect(component).toMatchSnapshot();
  });

});
