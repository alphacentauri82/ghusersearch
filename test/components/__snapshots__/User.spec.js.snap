// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: User should render 1`] = `
<a
  className="User_abw475 u-flex"
  href="/simon"
  onClick={[Function]}
>
  <div
    className="User_wrapAvatar_3afm6b"
  >
    <div
      className="Image_e296pg"
      style={
        Object {
          "minHeight": 60,
        }
      }
    >
      <img
        alt="simon's avatar"
        className="Avatar_1pcpsid u-imgResponsive"
        onLoad={[Function]}
        src="url&s=180"
        width={90}
      />
      <svg />
    </div>
  </div>
  <p
    className="User_username_rnfkwd"
  >
    simon
  </p>
</a>
`;
